omod/src/test/java/org/openmrs/module/pihcore/htmlformentry/HtmlFormComponentTest.java

```diff
@@ -45,9 +44,6 @@ public void beforeEachTest() throws Exception {
         executeDataSet("requiredDataTestDataset.xml");
         authenticate();
 
-        // load the API test bundle of MDS concepts
-        metadataDeployService.installBundle(Context.getRegisteredComponents(OrderEntryConceptsFromMetadataSharing.class).get(0));
```

Comment:
Do we just not need to load this for the tests, or am I missing the place where we start loading them via config instead?

Reply:
The tests still passed after removing, so...